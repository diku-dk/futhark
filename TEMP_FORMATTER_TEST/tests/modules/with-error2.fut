-- Refinement of a parametric with a non-parametric type is not OK.
-- ==
-- error: Cannot refine a type

module type has_t = { type t 'a }
module type has_t' = has_t with t = i32
