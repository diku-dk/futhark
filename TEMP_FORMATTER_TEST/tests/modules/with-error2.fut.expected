09d9d2b6989cdc3893433af6aac2b7ce
